parity/lib.rs

```diff
@@ -161,6 +192,8 @@ fn execute<Cr, Rr>(command: Execute, on_client_rq: Cr, on_updater_rq: Rr) -> Res
 	// 		they want
 	let logger = setup_log(&command.logger).expect("Logger is initialized only once; qed");
 
+	run_deadlock_detection_thread();
```

Comment:
I think it would be more verbose to have a compilation flag here 

```rust    
#[cfg(feature = "deadlock_detection")]
run_deadlock_detection_thread()
```